### DISCUSSION TITLE
Interest in including a `flake.nix` file for nightly/HEAD builds?

### DISCUSSION THREAD

**Discussion by PriceHiller** [neovide/neovide#2560] [General]
Hello! I noticed that Neovide doesn't provide a `flake.nix` (or other nix files) in its repository and that it points users to use what is provided via Nixpkgs.

It would be nice if Neovide did provide a `flake.nix` so Nix users could optionally point to HEAD builds of Neovide.

I understand you may not want to include a `flake.nix` because Nix is arcane and frankly *not* user friendly.

With that said, I'd be willing to create and maintain a `flake.nix` for Neovide going forward so you can pawn that problem off to me. Would you be open to including a `flake.nix` file if you have a dedicated maintainer for that aspect?

If you're still completely adverse to a `flake.nix`, no problem -- I totally get it.

**Comment by fredizzimo** [comment]
There was some talks about that previously on discord starting from here for example htps://discord.com/channels/896132176091955261/896263894086066216/1175370604728295424

I think the conclusion was that we could include a flake if someone creates it.

**Reply by PriceHiller** [reply]
> I think the conclusion was that we could include a flake if someone creates it.

Well, then by god I'll create it. I know there's some annoying edges with vendoring `skia` because of how Nix works. That's where most of the maintenance burden will crop up I bet.
